components/calendar/src/hebrew.rs

```diff
@@ -839,10 +847,15 @@ mod tests {
         for (case, f_date) in HEBREW_DATES.iter().zip(TEST_FIXED_DATE.iter()) {
             let date = Date::try_new_hebrew_date(case.year, case.month, case.day).unwrap();
             assert_eq!(
-                Hebrew::hebrew_from_fixed(RataDie::new(*f_date)),
+                BookHebrew::book_hebrew_from_fixed(RataDie::new(*f_date)),
                 date,
                 "{case:?}"
             );
         }
     }
+
+    #[test]
+    fn test_icu_bug() {
+        println!("{}", BookHebrew::days_in_hebrew_year(88369));
+    }
```

Comment:
Suggestion: name it `test_icu_bug_12345` (whatever the bug number is) and then make an `assert_eq!`